docs/cryptodoc/src/07_rng.rst

```diff
@@ -849,6 +849,48 @@ getrandom
       4. ``buf += got``
       5. ``len = len - got``
 
+External RNG Providers
+----------------------
+
+.. _rng/jitter_rng:
+
+JitterEntropy Library
+---------------------
+
+JitterEntropy is a library that provides a random number generator based on
+jitter of a CPU's execution time. For a detailed explanation of the RNG, its
+claimed security strength and assumptions, see the JitterEntropy documentation
+[#JitterEntropy_library]_.
```

Comment:
To fix CI: Remove `#`. Same in the bibliography. (The ref was once a footnote, I guess)